scenario/scenario.go

```diff
@@ -22,19 +22,23 @@ import (
 func CreateZRXForWETHSignedTestOrder(t *testing.T, ethClient *ethclient.Client, makerAddress, takerAddress common.Address, wethAmount *big.Int, zrxAmount *big.Int) *zeroex.SignedOrder {
 	// Create order
 	testOrder := &zeroex.Order{
-		MakerAddress:          makerAddress,
-		TakerAddress:          constants.NullAddress,
-		SenderAddress:         constants.NullAddress,
+		ChainID:         big.NewInt(constants.TestNetworkID),
+		ExchangeAddress: ethereum.NetworkIDToContractAddresses[constants.TestNetworkID].Exchange,
+		MakerAddress:    makerAddress,
+		TakerAddress:    constants.NullAddress,
+		SenderAddress:   constants.NullAddress,
+		// TODO(albrow): We should remove MakerFeeAssetData and TakerFeeAssetData after the DevUtils contract is fixed
```

Comment:
@fabioberger has the Ganache snapshot been updated to use the latest `DevUtils` contract? If so we should set the `MakerFeeAssetData` and `TakerFeeAssetData` to `NullBytes` and remove this `TODO`. I made this comment in a few other places as well.

Reply:
Fixed!